### ISSUE TITLE
Incorrect axis and cyclic symmetry determination

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-Py#1482]
### 1) Axis Symmetry

This was discovered with `C=[C]C=CCC`, which should have a symmetry number of 3 from the methyl rotor only. Using either of its resonance structures alone gives the correct result, but using the resonance hybrid (with bond orders of `[2, 1.5, 1.5, 1, 1]`) gives a symmetry number of 6. I've determined that the extra factor of 2 comes from `calculateAxisSymmetryNumber`, but had a hard time figuring out exactly what was going wrong.

### 2) Cyclic Symmetry

This was discovered with 'c1ccccc1[O]', which should have a symmetry number of 2. RMG currently does get the correct value using the resonance hybrid, but I'm not sure if it's for the right reason. I made a change to the set of resonance structures generated for this species, which changed the resonance hybrid and gave a new symmetry number of 12.

Looking into this a bit more, I noticed that the Kekule structure for this species also has a symmetry number of 12, even though the bonds are different. My current hypothesis is that vf2.feasible may not be properly comparing the atoms. It relies on having mapping and terminal attributes, which are assigned by the isomorphism function, which is not used in this case.

**Comment by goldmanm** [comment]
For the first problem, the solutions lies in this incorrect line of code `rmgpy/molecule.symmetry.py` written over 8 years ago in 11d2b27da66ceb5a5725ea91dbecad. Can you find what's wrong with it?

```python
# Do nothing if less than two cumulated bonds
if len(bonds) < 1: continue
```

Changing the `1` to a `2`, then causes errors estimating ethylene and formaldehyde, which we'll have to fix before resolving this issue.

**Comment by mliu49** [comment]
It seems like that was an intentional change, though the comment was not updated. See 8d72a1b33cf4aafbc4d3b037ba9cf9229fec4547.

**Comment by goldmanm** [comment]
fixed by #1487
